cmd/pyroscope/command/exec.go

```diff
@@ -0,0 +1,49 @@
+package command
+
+import (
+	"fmt"
+	"os"
+	goexec "os/exec"
+
+	"github.com/pyroscope-io/pyroscope/pkg/cli"
+	"github.com/pyroscope-io/pyroscope/pkg/config"
+	"github.com/pyroscope-io/pyroscope/pkg/exec"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newExecCmd(cfg *config.Exec) *cobra.Command {
+	execCmd := &cobra.Command{
+		Use:   "exec [flags] <args>",
+		Short: "starts a new process from arguments and profiles it",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if cfg.NoLogging {
+				logrus.SetLevel(logrus.PanicLevel)
+			} else if l, err := logrus.ParseLevel(cfg.LogLevel); err == nil {
+				logrus.SetLevel(l)
+			}
+			if len(args) == 0 || args[0] == "help" {
+				fmt.Println(gradientBanner())
+				fmt.Println(DefaultUsageFunc(cmd.Flags(), cmd))
```

Comment:
```suggestion
				_ = cmd.Help()
```

But I'm not sure if we should doing this at all: we have `--help` flag, it is sufficient.

Reply:
I agree with the second statement, it's probably redundant in this case, will remove it, unless @petethepig think there's a reason to keep it?

Reply:
Let's keep help subcommand, the easier it is for people to find help the better imo. From user perspective it's frustrating when you install some command and you don't know if it's `pyroscope -h`, `pyroscope -help`, `pyroscope --help` or `pyroscope help`. So why not just handle all of them.